tasks/tests/unit/test_process_flakes.py

```diff
@@ -400,3 +440,60 @@ def test_it_creates_flakes_expires(transactional_db):
         assert flake.fail_count == 1
         assert flake.start_date == old_time
         assert flake.end_date == new_time
+
+
+def test_it_creates_rollups(transactional_db):
+    traveller = time_machine.travel("1970-1-1T00:00:00Z")
+    traveller.start()
+    rs = RepoSimulator()
+    commits = []
+    c1 = rs.create_commit()
+    rs.merge(c1)
+    rs.add_test_instance(c1, outcome=TestInstance.Outcome.FAILURE.value)
+    rs.add_test_instance(c1, outcome=TestInstance.Outcome.FAILURE.value)
+
+    ProcessFlakesTask().run_impl(
+        None,
+        repo_id=rs.repo.repoid,
+        commit_id_list=[c1.commitid],
+        branch=rs.repo.branch,
+    )
+
+    traveller.stop()
```

Comment:
why is it checking for code coverage in a test file lol

Reply:
i actually don't know